src/components/formikComponents/formikDropdown.js

```diff
@@ -0,0 +1,435 @@
+import React, { useRef, useState } from 'react'
```

Comment:
can we change the current dropdown to be this formik dropdown, make it more separatable